package.json

```diff
@@ -43,7 +43,7 @@
     "test:init-e2e-tests": "python3 ./scripts/run_chain_tests.py"
   },
   "dependencies": {
-    "@polkadot/api": "^4.12.1",
+    "@polkadot/api": "beta",
```

Comment:
dq: any implications to using the beta channel / this mean we wouldn't have to update @polkadot/api ?

Reply:
I have the same question otherwise it looks good.

Reply:
> any implications

Probably best if we didn't rely on the beta for each weeks release. As of now I think we should just update `@polkadot/api` each monday to its main release, and if we have to do a beta update mid week to master to fix a bug then we do that. This way we could find potential issues per each main release. 

So i would say one major implication of beta would be making it harder to track changes and apply them per release.

Reply:
Ok, are you saying that this enabled this for testing and change to a specific release on Monday?

Reply:
> Ok, are you saying that this enabled this for testing/for it work and change to a specific release on Monday (when it's released)?

Yea exactly. This beta-version is just fixing a bug in polkadot-js for our master branch in regards to assets, but once monday comes we would switch back to the latest(official) release.